store/src/main/java/org/apache/rocketmq/store/timer/TimerMessageStore.java

```diff
@@ -81,7 +81,7 @@ public class TimerMessageStore {
     public static final int DAY_SECS = 24 * 3600;
     // The total days in the timer wheel when precision is 1000ms.
     // If the broker shutdown last more than the configured days, will cause message loss
-    public static final int TIMER_WHELL_TTL_DAY = 7;
+    public static final int TIMER_WHEEL_TTL_DAY = 7;
```

Comment:
To be precise,  Rename it to "Timing Wheels". See  http://www.cs.columbia.edu/~nahum/w6998/papers/sosp87-timing-wheels.pdf